.github/workflows/build.yml

```diff
@@ -54,6 +54,8 @@ jobs:
       shell: bash
       run: |
         unset M2_HOME
+        mkdir -p ~/.m2
+        echo "<settings><mirrors><mirror><mirrorOf>central</mirrorOf><name>central</name><url>https://repo.maven.apache.org/maven2/</url></mirror></mirrors></settings>" > ~/.m2/settings.xml # reqiored by Maven < 3.2.3
```

Comment:
At the end is a typo: reqiored

Reply:
Also, integration tests does not seem to use this conf, see test failures